apps/web/modules/ee/teams/team-list/components/manage-team-button.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "@/modules/ui/components/button";
+import { TooltipRenderer } from "@/modules/ui/components/tooltip";
+import { useTranslations } from "next-intl";
+
+interface ManageTeamButtonProps {
+  onClick: () => void;
+  disabled?: boolean;
```

Comment:
Can we mark this as required, as for both times when this button is used, we have some value for `disabled`